src/Parsers.jl

```diff
@@ -21,7 +21,8 @@ export Header, Headers,
        find_end_of_header, find_end_of_line, find_end_of_trailer,
        parse_status_line!, parse_request_line!, parse_header_field,
        parse_chunk_size,
-       ParseError
+       ParseError,
+       lceq
```

Comment:
This needs to stay in Messages.jl

The spec for the Parser layer is clear on this (see line 9):
> The Parser does not interpret the Message Headers. It is beyond the scope of the Parser to deal with ... or case normalization.

Reply:
Moved.